src/jibri_tracker.ts

```diff
@@ -54,21 +60,41 @@ export class JibriTracker {
     }
 
     async track(state: JibriState): Promise<boolean> {
-        const key = `jibri:idle:${state.jibriId}`;
-        if (
-            state.status.busyStatus === JibriStatusState.Idle &&
-            state.status.health.healthStatus === JibriHealthState.Healthy
-        ) {
-            const result = await this.redisClient.set(key, 1, 'ex', JibriTracker.idleTTL);
-            if (result !== 'OK') {
-                throw new Error(`unable to set ${key}`);
-            }
-            return true;
+        let group = 'default';
+        // pull the group from metadata if provided
+        if (state.metadata && state.metadata.group) {
```

Comment:
I have a couple of questions. Why are you optionally handling this? I wonder because if it has to do with the webhook call and versioning then we should chat. 

If it is indeed something we want to optionally have then I think it's better to put it explicitly in the interface as optional using ? or something I think.